apps/voting/app/src/script.js

```diff
@@ -262,19 +260,13 @@ function loadVoteSettings() {
 
 function loadTokenDecimals(tokenContract) {
   return new Promise((resolve, reject) => {
-    tokenContract
-      .decimals()
-      .first()
-      .subscribe(resolve, reject)
+    tokenContract.decimals().subscribe(resolve, reject)
   })
 }
 
 function loadTokenSymbol(tokenContract) {
   return new Promise((resolve, reject) => {
-    tokenContract
-      .symbol()
-      .first()
-      .subscribe(resolve, reject)
+    tokenContract.symbol().subscribe(resolve, reject)
```

Comment:
Instead of wrapping with a promise, we can call `.toPromise`, i.e.     `return tokenContract.symbol().toPromise()`

Reply:
Much better 👌